browser/ui/views/sidebar/sidebar_container_view_browsertest.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/brave_browser.h"
+#include "brave/browser/ui/sidebar/sidebar_controller.h"
+#include "brave/browser/ui/sidebar/sidebar_service_factory.h"
+#include "brave/browser/ui/views/frame/brave_browser_view.h"
+#include "brave/browser/ui/views/sidebar/sidebar_button_view.h"
+#include "brave/browser/ui/views/sidebar/sidebar_container_view.h"
+#include "brave/components/constants/pref_names.h"
+#include "brave/components/sidebar/sidebar_item.h"
+#include "brave/components/sidebar/sidebar_service.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/views/frame/toolbar_button_provider.h"
+#include "chrome/browser/ui/views/toolbar/side_panel_toolbar_button.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "components/prefs/pref_service.h"
+#include "content/public/test/browser_test.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+class SidebarContainerViewBrowserTest : public InProcessBrowserTest {
+ public:
+  SidebarContainerViewBrowserTest() {}
+  SidebarContainerViewBrowserTest(const SidebarContainerViewBrowserTest&) =
+      delete;
+  SidebarContainerViewBrowserTest& operator=(
+      const SidebarContainerViewBrowserTest&) = delete;
+
```

Comment:
nit: d'tor for a child class

```suggestion
  ~SidebarContainerViewBrowserTest() override = default;
```